pkg/cli/cli.go

```diff
@@ -487,6 +488,138 @@ func (c *CLIRunner) GetTopics(ctx context.Context, full bool) error {
 	return nil
 }
 
+// GetUnderReplicatedPartitions fetch all under replicated partitions and prints out a summary.
+func (c *CLIRunner) GetUnderReplicatedPartitions(ctx context.Context, topics []string) error {
+	c.startSpinner()
+	urpsFound := false
+
+	filterTopics := make(map[string]bool)
```

Comment:
In this file we usually put all the business logic into a method on the adminClient. This allows us to have different implementations for Kafka and Zookeeper. I think this would lead to some strange errors if someone tried to use this with Zookeeper right now. If we move this into a method on the adminClient we have ways to mark methods as unsupported and there will be consistent error messaging around that

Reply:
So it would look something like [this](https://github.com/segmentio/topicctl/blob/7ac22690645e4141e6cd134abe68eb67218b8318/pkg/admin/brokerclient.go#L242) for Kafka. The Zookeeper implementation would return nil and we'd find some way to [mark this as unsupported](https://github.com/segmentio/topicctl/blob/7ac22690645e4141e6cd134abe68eb67218b8318/pkg/admin/zkclient.go#L760) using our `GetSupportedFeatures`